### ISSUE TITLE
Cannot read property 'startTimestamp' of null

### ISSUE THREAD

**Issue by vasanthdeveloper** [iCrawl/discord-vscode#371]
Even after starting Discord first, and then opening VSCode, I still get the error:
```
Discord Presence activated!
Logging into RPC.
Successfully connected to Discord.
Sending activity to Discord.
TypeError: Cannot read property 'startTimestamp' of null
```
This happens on my Arch Linux workstation where I usually code. I am using the KDE Plasma desktop with VSCode installed from the following AUR package: https://aur.archlinux.org/packages/visual-studio-code-bin/. The extension behaves as it should on my Windows 10 machine, but the error occurs only on my Linux workstation.

Here are the steps to reproduce it:
1. Log in to the computer
2. Start Discord App (installed from official package repositories)
3. Start Visual Studio Code with Discord Presence enabled.
    - Expectation: My status shows up on Discord.
    - Result: I get the following error: `TypeError: Cannot read property 'startTimestamp' of null` on the output tab of VSCode.

![telegram](https://user-images.githubusercontent.com/24322511/64854613-a7508f80-d63b-11e9-9f6a-857ed4cc4a8c.png)

This prevents from showing the status on Discord.

**Comment by iCrawl** [comment]
I don't really see how this could happen, esp not when it works on windows.

Code-wise I never access the property without checking before that the object exists.

**Comment by vasanthdeveloper** [comment]
I have sent you the screenshot. How can I help you regarding this issue?

**Comment by iCrawl** [comment]
- Clone the repo
- Uninstall the extension
- Change the `logger.log` to `console.log`
https://github.com/iCrawl/discord-vscode/blob/394ad217e03c1ff9cd03f676faa008936b763afa/src/extension.ts#L113
- Press F5, I guess? That should launch the extension in debug mode.

**Comment by vasanthdeveloper** [comment]
Though I have worked with JavaScript & TypeScript in the past, I have never written a VSCode extension. I cloned the repository, uninstalled the extension, and modified the `extension.ts` file as instructed. What should I do now?

**Comment by iCrawl** [comment]
Do you have Node.js installed?

If so thats a prerequisite, after that you run `npm install`.
After that you open the project inside of VSCode and press F5 (which executes the debugger)

**Comment by vasanthdeveloper** [comment]
Okay, yes I am also a JavaScript developer :)

**Comment by vasanthdeveloper** [comment]
After doing a bit of _trail-and-error_, I found out that installing the Extension from Marketplace would not show any error as well as not show the status on my Discord profile. (Meaning, no errors but doesn't work as expected). Strangely I cloned the git repository using:
```
git clone git@github.com:iCrawl/discord-vscode.git $HOME/.vscode/extensions
```
Opened VSCode inside the freshly cloned project and pressed F5 (to debug the extension). In this case, it works!

After repeating this a few times, I assume there are some permission problems when VSCode installs the extension, rather than me cloning it. Another possibility could be that JavaScript code that VSCode is pulling from the Marketplace has something wrong in it when I compile the TypeScript code into JavaScript using the commands:
```
yarn install
yarn run build
```
Visual Studio Code's debugger does not report any errors or warnings, and the extension works as expected.
